src/database/index.tsx

```diff
@@ -152,30 +177,6 @@ export const getBaselineScenarioRef = async () => {
   return baselineScenario.ref;
 };
 
-export const getFacilities = async (): Promise<Array<Facility> | null> => {
```

Comment:
Ignore:  I just moved this such that the `scenario` and `facility` related persistence methods were grouped together